chainer/utils/argument.py

```diff
@@ -0,0 +1,13 @@
+def check_unexpected_kwargs(kwargs, **unexpected):
+    for key, message in unexpected.items():
+        if key in kwargs:
+            raise ValueError(message)
+
+
+def parse_kwargs(kwargs, *name_and_values):
+    values = [kwargs.pop(name, default_value)
+              for name, default_value in name_and_values]
+    if kwargs:
+        args = ', '.join(["'%s'" % arg for arg in kwargs.keys()])
+        raise TypeError('got an unexpected keyword argument %s' % args)
```

Comment:
`an unexpected keyword argument` -> `unexpected keyword argument(s)` (or write more to change the wording for a single arg and multiple args)